external-adapter/test/config.test.ts

```diff
@@ -1,6 +1,7 @@
 import { expect } from 'chai'
```

Comment:
This test does not make much sense anymore at this location, we should move it to the place where `API_KEY` env is set, which is now the `external-adapter` package.

Reply:
That's already where this one is moved to :smile: